pom.xml

```diff
@@ -0,0 +1,159 @@
+ ~ Copyright (c) 2016 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~      http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+-->
+
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.wso2.extension.siddhi</groupId>
+    <artifactId>org.wso2.extension.siddhi.window.uniquelength</artifactId>
+    <version>1.0.0</version>
+
+    <packaging>bundle</packaging>
+    <name>WSO2 Siddhi - Window Library For Unique Length</name>
+    <url>http://wso2.org</url>
+    <dependencies>
+        <dependency>
+            <groupId>org.wso2.siddhi</groupId>
+            <artifactId>siddhi-core</artifactId>
+            <version>${version.org.wso2.siddhi}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.siddhi</groupId>
+            <artifactId>siddhi-query-api</artifactId>
+            <version>${version.org.wso2.siddhi}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.log4j.wso2</groupId>
+            <artifactId>log4j</artifactId>
+            <version>${version.org.apache.log4j.wso2}</version>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>${version.junit}</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+    <properties>
+        <version.org.wso2.siddhi>3.1.0-alpha</version.org.wso2.siddhi>
+        <version.org.apache.log4j.wso2>1.2.17.wso2v1</version.org.apache.log4j.wso2>
+        <version.junit>4.10</version.junit>
+    </properties>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>org.wso2.extension.siddhi.window.uniquelength</Bundle-SymbolicName>
+                        <Bundle-Name>org.wso2.extension.siddhi.window.uniquelength</Bundle-Name>
+                        <Export-Package>
+                            org.wso2.extension.siddhi.window.uniquelength,
+                            org.wso2.extension.siddhi.window.uniquelength.*
```

Comment:
Export packages should have version specified as a best practice. Please fix these type of issues throughout in all your PR's.